### PR TITLE
137 change user model style

### PR DESCRIPTION
**Pull request by Emalsha**
Style user model PR list.
Add custom scroll bar.

### CODE REVIEW

**Comment by Emalsha** [comment]
#137 Fixes

**Comment by iamdarshshah** [comment]
@Emalsha This really looks great and everything is working as expected. Just need one more change, please check the icons for `open`, `closed` & `merged` states on GitHub and update the icons as required.

Other than that, It looks good to me! Thanks @Emalsha for taking out some time for this issue 👍

**Comment by Emalsha** [comment]
Done 👍👍👍

File: src/components/Profile/Profile.styles.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { makeStyles } from '@material-ui/core';
+import { makeStyles, Hidden } from '@material-ui/core';
```
Comment (iamdarshshah) [inline src/components/Profile/Profile.styles.js:1]:
```suggestion
import { makeStyles } from '@material-ui/core';
```
Are you making use of `Hidden`? If not then please remove it.

Comment (Emalsha) [reply src/components/Profile/Profile.styles.js]: Removed ✌

**Review by iamdarshshah — APPROVED** [review]
Perfect! Thanks @Emalsha for your efforts 👍

**Comment by iamdarshshah** [comment]
Closes #137 🎉

**Pull request merged**